interactions/client/utils/input_utils.py

```diff
@@ -25,9 +26,11 @@
     import msgspec.json as json
 
     def enc_hook(obj: Any) -> int:
-        # msgspec doesnt support IntFlags
+        # msgspec doesnt support IntFlags or interactions.Snowflakes
         if isinstance(obj, IntFlag):
```

Comment:
You can pass a tuple of types to check, rather than duplicating the entire if block